lib/tapioca/cli.rb

```diff
@@ -302,9 +302,9 @@ def check_shims
     end
 
     desc "annotations", "Pull gem annotations from a central RBI repository"
-    option :repo_uri, type: :string, desc: "Repository URI to pull annotations from", default: CENTRAL_REPO_ROOT_URI
+    option :repo_uri, type: :array, desc: "Repository URI to pull annotations from", default: [CENTRAL_REPO_ROOT_URI]
```

Comment:
Since it accepts an array, could we change the option name to plural (repo_uris) and also update the description?

Reply:
I renamed it as `--repos` (https://github.com/Shopify/tapioca/pull/955/commits/98663d1a602a8343f0f880aebad6a92ad101c7ad) let me know what you think.